Kwf/Assets/Dependency/File.php

```diff
@@ -168,7 +168,7 @@ protected function _getComponentCssClass($prefix)
         return null;
     }
 
-    private static function _getAbsolutePath($path)
+    public static function getAbsolutePath($path)
```

Comment:
now that it is public we should think about a good name: calculateAbsolutePath, resolveAbsolutePath

get (especally in Dependency_File) is missleading